compassion_nordic_accounting/wizards/load_mandate_wizard.py

```diff
@@ -43,13 +43,20 @@ class LoadMandateWizard(models.Model):
     kid = fields.Char(help="kundeidentifikasjon, is the payment reference for a specific partner")
     old_mandate_state = fields.Char()
     current_mandate_state = fields.Char(compute="_compute_all", store=True)
+    is_cancelled = fields.Boolean(default=False)
 
     @api.depends('mandate_id')
     def _compute_all(self):
         for rec in self:
             rec.partner_id = rec.mandate_id.partner_id if rec.mandate_id else rec.partner_id
             rec.company_id = rec.mandate_id.company_id
-            rec.current_mandate_state = rec.mandate_id.state if rec.mandate_id else rec.old_mandate_state
+            if rec.mandate_id:
+                current_state = rec.mandate_id.state
+            elif rec.is_cancelled:
+                current_state = "Cancelled"
```

Comment:
Design suggestion as discussed
```suggestion
                current_state = "cancel"
```